### DISCUSSION TITLE
A warning appears when running "pip install -r requirements.txt"

### DISCUSSION THREAD

**Discussion by shisenglin** [ultralytics/yolov5#7727] [Q&A]
![0d67f3701c3f178c07b2374907b2d43](https://user-images.githubusercontent.com/101716952/167259940-176d97f8-e8a0-407c-9920-4c762312c012.png)
When I run "pip install -r requirements.txt" I get the error in the picture, how can I fix it?

**Comment by glenn-jocher** [comment]
@shisenglin I don't see any errors in your screenshot, these are simply warning and can probably be ignored.
